docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -31,6 +31,8 @@ There you can find links to upgrade notes for other versions too.
     - [`Version20190121094400`](https://github.com/shopsys/shopsys/blob/master/packages/framework/src/Migrations/Version20190121094400.php <!--- TODO: change to released version instead of master -->
 
 ### Application
+- modify ErrorController to be more descriptive about error caused by active TEST environment ([#701](https://github.com/shopsys/shopsys/pull/701))
+    - modify `ErrorController::createUnableToResolveDomainResponse()` by these [changes](https://github.com/shopsys/shopsys/pull/701/files#diff-0b1aecbf82624ce474ca3cb8bd75811c).
```

Comment:
You have here too big indentation

Reply:
no its not, its nested because it belongs to it.

Reply:
Okay, in that case please updated line above so it does not sound as instruction. @MattCzerner